components/Navigation/Navigation.css

```diff
@@ -0,0 +1,28 @@
+
+.navigation{
+    display:flex;
+    justify-content:space-between;
+    align-items:center;
+    width:98%;
+    height:3rem;
+    background:var(--secondary-color);
+    
+    border-radius:5px;
+    font-weight: bold;
+    font-size: 1.1rem;
+    padding: 0 1rem;
+    
+  }
+  .navigation_items{
+    float:right;
+    display:flex;
+}
```

Comment:
I thinks float is not good practice with flex. Other than that you can use align and justify properties to position them.

Reply:
Okay,will try to imrove it.Thankyou

Reply:
you may also try `auto-margin`

Reply:
> you may also try `auto-margin`

okay..Thank for the suggestion